packages/api-plugin-fulfillment/src/mutations/createFulfillmentMethod.test.js

```diff
@@ -0,0 +1,72 @@
+import mockCollection from "@reactioncommerce/api-utils/tests/mockCollection.js";
+import mockContext from "@reactioncommerce/api-utils/tests/mockContext.js";
+import createFulfillmentMethodMutation from "./createFulfillmentMethod.js";
+
+mockContext.validatePermissions = jest.fn().mockName("validatePermissions");
+mockContext.collections.Fulfillment = mockCollection("Fulfillment");
+mockContext.validatePermissions.mockReturnValueOnce(Promise.resolve(null));
+
+test("throws if required fields are not supplied", async () => {
+  const fulfillmentMethodInput = {
+    method: {
+      shopId: "SHOP_ID",
+      cost: 99,
+      fulfillmentMethod: "flatRates",
+      displayMessageMethod: "Sample display message"
+    },
+    fulfillmentTypeId: "fulfillment123",
+    shopId: "SHOP_ID"
+  };
+
+  await expect(createFulfillmentMethodMutation(mockContext, fulfillmentMethodInput)).rejects.toThrowErrorMatchingSnapshot();
```

Comment:
@aldeed @zenweasel @tedraykov @vanpho93 
Brent has the below suggestion in the old PR [link](https://github.com/reactioncommerce/reaction/pull/6570#discussion_r1004018632) 

> Just my opinion but I think using snapshots are fine for frontend testing but for backend PR's they make it unnecessarily brittle. Open to second opinions on that.

I shall go ahead and make this change unless someone thinks otherwise.

Reply:
I often use the `to*InlineSnaphot` variants with good success. Basically just saves you from needing to type out the objects. No strong preference, though

Reply:
As we are not including new Jest specific entries, did a comparison against locally declared variable.

Reply:
Marking resolved as changes were pushed to branch